ChattoAdditions/sources/Chat Items/CompoundMessage/CompoundMessagePresenter.swift

```diff
@@ -319,6 +322,9 @@ open class CompoundMessagePresenter<ViewModelBuilderT, InteractionHandlerT>
                 self?.handleContentTransferStatusUpdate()
             })
         }
+        if let contentLoading = presenter as? MessageContentLoadingPresenterProtocol {
```

Comment:
I was thinking if we could somehow get rid of this casting. What do you think about this:
1. `contentLoadingPresenterSetup` doesn't seem to be injected in both `Chatto` and main application. Do you think we can just get rid of it 😅
2. Since we are calling it on the `createContentPresenter`, we could potentially add a method on `MessageContentPresenterProtocol` named more generically. (For example `onPresenterSetup`).

Reply:
Replaced with more generic `contentPresenterSetup` version